packages/utils/src/math/interpolate.js

```diff
@@ -74,7 +75,7 @@ export function interpolate(
 	if (isColor(outputRange1) && isColor(outputRange2)) {
 		const mixAmount = interpolate(input, inputRange, [0, 100]);
 		return colorize
-			.mix(outputRange1, outputRange2, mixAmount)
+			.mix(outputRange1.toString(), outputRange2.toString(), mixAmount)
```

Comment:
The `ColorInput` type only accepts strings, not numbers (even though it works fine if you pass a number). It might be worth updating upstream?

Reply:
Maybe. `Infinity`, `NaN` and others are in the `Number` type. You can certainly pass bad strings in the string type as well although [template literal types in TypeScript 4.1](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1-beta/#template-literal-types) may allow some interesting types.